src/constants.ts

```diff
@@ -86,3 +86,5 @@ var resources: any = {
 };
 
 _.set(window, 'OPENSHIFT_CONSTANTS.CATALOG_HELP_RESOURCES', resources);
+
+_.set(window, 'OPENSHIFT_CONSTANTS.NEW_PROJECT_MESSAGE', "Contact your system administrator at projectname@example.com to request a project.");
```

Comment:
@jeff-phillips-18 This is customized in OpenShift master-config.yaml. We should use that message.

https://github.com/openshift/origin-web-console/blob/master/app/scripts/controllers/projects.js#L154-L163